pkgs/applications/blockchains/alfis/default.nix

```diff
@@ -27,10 +26,16 @@ rustPlatform.buildRustPackage rec {
     "--skip=dns::client::tests::test_udp_client"
   ];
 
-  nativeBuildInputs = [ pkg-config ];
+  nativeBuildInputs = [ pkg-config makeWrapper ];
   buildInputs = lib.optional (withGui && stdenv.isLinux) webkitgtk
     ++ lib.optionals (withGui && stdenv.isDarwin) [ Cocoa WebKit ];
 
+  postInstall = lib.optionalString (withGui && stdenv.isLinux) ''
+    cp $out/bin/alfis{,_unwrapped}
+    makeWrapper $out/bin/alfis{_unwrapped,} \
```

Comment:
```suggestion
    makeWrapper $out/bin/alfis \
```
Isn't the auto generated unwrapped name enough?

Reply:
I... don't know. This suggestion fails on my machine...

Reply:
How does it fail? I need more details if I should provide help.
makeWrapper renames the original program to .alfis-wrapped if you only provide the binary name.

Reply:
```
       > glibPreInstallPhase
       > installing
       > Executing cargoInstallHook
       > dirname: unrecognized option '--prefix'
       > Try 'dirname --help' for more information.
       > mkdir: cannot create directory '': No such file or directory
```